ls1intum/ArTEMiS
# Feature/student can now register for a new course

### Checklist

### Motivation and Context
A student should be able to register for a course. this decreases the maintenance of a course.

### Description
added a button to register for a new course from the student dashboard.

### Steps for Testing

1. Log in to ArTEMiS
2. Navigate to Course Administration
3. Make sure that a course exsists which has a start date before today and an end date after today
4. Login as a student
5. go to the student dashboard and click the `register for course` button in the top right corner
6. select course
7. check if the process was done sucessfully 

### Screenshots
![image](https://user-images.githubusercontent.com/10998002/53958810-440d0000-40e2-11e9-80f7-9b7425dc358f.png)
![image](https://user-images.githubusercontent.com/10998002/53958825-4b340e00-40e2-11e9-85dd-3095f86433a4.png)
![image](https://user-images.githubusercontent.com/10998002/53958839-50915880-40e2-11e9-9bfc-7ad5629fcf70.png)
![image](https://user-images.githubusercontent.com/10998002/53958860-5d15b100-40e2-11e9-9894-f53800544518.png)
